templates/scenarios/sso-tab-js/teamsfx/app.yml

```diff
@@ -22,15 +22,15 @@ provision:
 deploy:
   - uses: npm/command # Run npm command
     with:
-      workingDirectory: ./tabs
+      workingDirectory: .
       args: install
   - uses: npm/command # Run npm command
     with:
-      workingDirectory: ./tabs
+      workingDirectory: .
```

Comment:
In deploy, it uses "./" for current folder.  Should we use one style?

Reply:
thanks, updated